sources/packages/web/src/services/http/common/HttpBaseClient.ts

```diff
@@ -62,11 +62,23 @@ export default abstract class HttpBaseClient {
     }
   }
 
-  protected async patchCall<T>(url: string, payload: T): Promise<void> {
+  /**
+   * Executes a HTTP request using a PATCH verb including the authentication token.
+   * @param url API endpoint URI.
+   * @param payload data to be sent.
+   * @param suppressErrorHandler optionally skip the global error handling.
+   */
+  protected async patchCall<T>(
+    url: string,
+    payload: T,
+    suppressErrorHandler = false,
```

Comment:
Not a part of the PR. suppressErrorHandler can also be added to get and post common method.

Reply:
Yes, we can add on demand.